packages/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -38,6 +38,9 @@ export interface AccordionTitleProps
   /** If at least one panel needs to stay active and this title does not correspond to the last active one. */
   canBeCollapsed?: boolean
 
+  /** Disabled - does not react on click events */
```

Comment:
```suggestion
  /** Disables clicks interaction and changes accessibility behavior. */
```

Is it better? If no, let's add at least trailing dot 🐱 

```suggestion
  /** Disabled - does not react on click events. */
```